spec/javascripts/angular/services/variable_interpolator_spec.js

```diff
@@ -8,7 +8,8 @@ describe('VariableInterpolator', function() {
     this.availableFields = {
       name: "field name",
       quantile: "0.75",
-      server: "http://localhost:8080/metrics"
+      server: "http://localhost:8080/metrics",
+      job: "prometheus-stream"
```

Comment:
Is it valid to keep a trailing comma in the last line in JS? Then the diffs become cleaner.